src/Servers/Kestrel/samples/Http3SampleApp/Http3SampleApp.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <!-- Turn off warnings about using preview features -->
+    <NoWarn>CA2252;$(NoWarn)</NoWarn>
```

Comment:
Since this is a sample app, it should probably enable preview features, rather than disabling the warning, [see the design document](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#enabling-a-preview-feature).

```suggestion
    <!-- Turn on preview features so we can use Http3 -->
    <EnablePreviewFeatures>True</EnablePreviewFeatures>
```